src/SDKs/Monitor/Management.Monitor/Generated/Models/EventData.cs

```diff
@@ -128,175 +128,172 @@ public EventData(EventLevel level, System.DateTime eventTimestamp, System.DateTi
         /// <summary>
         /// </summary>
         [JsonProperty(PropertyName = "authorization")]
-        public SenderAuthorization Authorization { get; set; }
+        public SenderAuthorization Authorization { get; private set; }
```

Comment:
All these are mainly cosmetic changes, but they do have some consequences in the rest of the code.